### ISSUE TITLE
Failure to load code if profiling on & using racket/class

### ISSUE THREAD

**Issue by zenspider** [racket/racket#1551]
I've reduced it down to:

```
#lang racket/base ;; needs to have "Debugging and Profiling" set
(require racket/class)

(define register%
  (class object%
    (super-new)

    (define/public (equal-to? other recur)
      (and #t))))
```

gives:

```
[...]share/pkgs/errortrace-lib/errortrace/stacktrace.rkt:199:20: errortrace:
  unrecognized (non-top-level) expression form: (#%expression (quote #t))
```

**Comment by zenspider** [comment]
I can also reproduce this error using emacs + racket-mode with `C-c C-l`... that's apparently running this in the repl:

```
> ,run ("myfile.rkt") 2048 t profile nil
```

**Comment by rfindler** [comment]
Thanks for the report, but this appears to have been fixed since 6.7. Probably @stamourv 's commit: https://github.com/racket/errortrace/commit/f332fd9d4c75d57c40b78e60c9c05f8499a70cd1
